### PR TITLE
FVM 031 Services Page

### PR DESCRIPTION
**Pull request by jpickel**

### CODE REVIEW

File: wp-content/themes/fvm/functions.php [hunk 1]
```diff
@@ -408,4 +408,13 @@ function mytheme_comment($comment, $args, $depth) {
 <?php
         }
 
+function register_styles() {
```
Comment (jpickel) [inline wp-content/themes/fvm/functions.php]: Is there a problem with just adding this CSS file as an import in the main style.css?

File: wp-content/themes/fvm/loop-lp.php [hunk 2]
```diff
@@ -285,6 +285,61 @@ function AoProcessForm(formelement) {
 	                </a>
 	                    <?php wp_reset_postdata(); ?>
 	                <?php endif; ?>
+
+	            <?php } elseif(get_row_layout() == "lp_supporting_case_study") { ?>
+	                <?php $post_object = get_sub_field('case_study_object'); if( $post_object ): $post = $post_object; setup_postdata( $post ); ?>
+	                <?php 
+	                	$text_color = get_sub_field('text_color');
```
Comment (jpickel) [inline wp-content/themes/fvm/loop-lp.php]: Don't save custom fields as variables, unless it's needed for a function. Use `<?php the_field(''); ?>` and `<?php the_sub_field(''); ?>` instead.

Comment (jpickel) [reply wp-content/themes/fvm/loop-lp.php]: This comment goes for other places where this is occurring as well.

File: wp-content/themes/fvm/loop-services.php [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+		
+		<section class="services-section bg white flex">
+
+		    <?php while ( have_rows('services') ) : the_row(); ?>
+				<?php 
+					$service_title = get_sub_field('service_title');
+					$service_page_link = get_sub_field('service_page_link');
+					$service_content = get_sub_field('service_content');
+				 ?>
+				<div class="col-md-3 service">
```
Comment (jpickel) [inline wp-content/themes/fvm/loop-services.php]: Looks like we need to add 10px of padding to what's already there above these blocks and the grey bg above it.

File: wp-content/themes/fvm/loop-services.php [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+					<h3 class="service__title"><a class ="service__link" href="<?php echo $service_page_link; ?>" class="service__link"><?php echo $service_title; ?></a></h3>
```
Comment (jpickel) [inline wp-content/themes/fvm/loop-services.php]: Click state changes color and has odd underline. Check with Russell on behavior.

File: wp-content/themes/fvm/loop-services.php [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+						<?php echo $service_content; ?>
+					</p>
+				</div>
+
+		    <?php endwhile; ?>
+
+		</section>
+
+	<?php endif; ?>
+	<section class="supporting-content">
```
Comment (jpickel) [inline wp-content/themes/fvm/loop-services.php]: Russell & Tom wanted to add in the hover effect that's seen on the home page to these blocks. We should be able to reuse the code that's on the home page. Take the same content that displays from the respective shaped box.

File: wp-content/themes/fvm/loop-services.php (hunk 3 above)
Comment (jpickel) [inline wp-content/themes/fvm/loop-services.php]: These boxes don't respond responsively. Same as on the individual service page.

File: wp-content/themes/fvm/css/services.css [hunk 6]
```diff
@@ -0,0 +1,120 @@
+.services #header .block .inner.orange {
```
Comment (jpickel) [inline wp-content/themes/fvm/css/services.css]: There are some general responsive issues. Also, the header changes back to orange on smaller screens.

File: wp-content/themes/fvm/functions.php (hunk 1 above)
Comment (stevenrayesky) [reply wp-content/themes/fvm/functions.php]: I don't think there is a problem. Wordpress recommends wp_enqueue_style() for best practice, but either can be used.

File: wp-content/themes/fvm/loop-lp.php (hunk 2 above)
Comment (stevenrayesky) [reply wp-content/themes/fvm/loop-lp.php]: I really like having the variables declared at the beginning of a page. If I return to a page, I immediately know what dynamic content is on the page and where to find it. If I just use `<?php the_sub_field(''); ?>` I don't necessarily know where on the page these variables are or how many variables exist. But if you are really adamant that we do not use them I can remove them.

**Comment by jpickel** [comment]
@stevenrayesky Can you forward me this image that's on your local machine?

http://localhost:8888/thinkfvm/wp-content/uploads/services-marquee.jpg

File: wp-content/themes/fvm/functions.php (hunk 1 above)
Comment (jpickel) [reply wp-content/themes/fvm/functions.php]: This is fine then

File: wp-content/themes/fvm/loop-lp.php (hunk 2 above)
Comment (jpickel) [reply wp-content/themes/fvm/loop-lp.php]: This is fine then. Maybe we can come up with standards on how to apply custom field values. Like when to use ACF's method and when to use variables.

**Pull request merged**